test/integration_test.py

```diff
@@ -147,30 +147,35 @@ def tearDown(self):
     def test(self):
         if config.deployment_stage != 'prod':
             try:
-                self._test_indexing()
+                log.info('Starting test using test name, %s ...', self.test_name)
+                azul_client = self.azul_client
```

Comment:
Wondering if there is a reason why this was not inline?

Reply:
Please go ahead and inline it.